caffe2/CMakeLists.txt

```diff
@@ -573,6 +573,7 @@ if (NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
       ${TORCH_SRC_DIR}/csrc/api/src/nn/options/linear.cpp
       ${TORCH_SRC_DIR}/csrc/api/src/nn/options/normalization.cpp
       ${TORCH_SRC_DIR}/csrc/api/src/nn/options/pooling.cpp
+      ${TORCH_SRC_DIR}/csrc/api/src/nn/options/vision.cpp
```

Comment:
I think we might not need to create the `torch/csrc/api/src/nn/options/vision.cpp` file, and having the header file should be sufficient :D